.circleci/config.yml

```diff
@@ -825,7 +918,18 @@ workflows:
             branches:
               only: master
               # touch: 1676417203
-      - deploy-to-gar:
+      - deploy-postgres:
+          requires:
+            - mysql-e2e-tests
+            - postgres-e2e-tests
+            - spanner-e2e-tests
+          filters:
+            tags:
+              only: /.*/
+            branches:
+              only: master
+              # touch: 1676417203
```

Comment:
I don't think we need this.

Reply:
Yeah, I've wondered what that is... I'll take it out. Wondered if it had some historical significance. Looked at the git blame but didn't offer much and was 2 years ago. Probably useless at this point